### ISSUE TITLE
Hi, I am attaching one more file which I have generated from Captivate but I am not able to implement resume option for this file. Please guide me.

### ISSUE THREAD

**Issue by Kopal-LMS** [dhodges47/SCORM-LearningManagementSystem#12]
[SCORMCaptivate8.zip](https://github.com/dhodges47/SCORM-LearningManagementSystem/files/4810175/SCORMCaptivate8.zip)

While using the file, I am not getting any results in below tables-
select * from [dbo].[cmi_interactions_objectives]
select * from [dbo].[cmi_interactions_correct_responses]
select * from [dbo].[cmi_comment_from_learner]
select * from [dbo].[cmi_comment_from_lms]
select * from [dbo].[cmi_objectives]

Please give the direction, how I get response in this tables.
Thank you so much.

**Comment by dhodges47** [comment]
Interactions are only stored if your course has an assessment test.
It doesn't look like the course you uploaded has a test.
Comments from learner wouldn't be stored because your course hasn't got any pace where the learner can enter comments.
I've never seen comment_from_lms used.
Objectives would only be filled in if Captivate gave you an option to enter course objectives. I haven't seen cmi_objectives used very often.
In Summary, based on the course you uploaded, none of the fields you listed need to have data in them. And many SCORM authoring tools don't use all the possible fields in all the possible tables any way.
